sites/public/pages/account/edit.tsx

```diff
@@ -0,0 +1,349 @@
+                error={errors.firstName}
+                errorMessage={t("errors.firstNameError")}
+                register={register}
+                defaultValue={profile ? profile.firstName : null}
+              />
+
+              <Field
+                name="middleName"
+                placeholder={`${t("application.name.middleNameOptional")}`}
+                register={register}
+                defaultValue={profile ? profile.middleName : null}
+              />
+
+              <Field
+                name="lastName"
+                placeholder={`${t("application.name.lastName")}`}
+                error={errors.lastName}
+                errorMessage={t("errors.lastNameError")}
+                register={register}
+                defaultValue={profile ? profile.lastName : null}
+              />
+              <div className="text-center">
+                <Button
+                  onClick={() => {
+                    //
+                  }}
+                  className={"items-center"}
+                >
+                  {t("account.settings.update")}
+                </Button>
+              </div>
+            </div>
+          </Form>
+          <Form id="update-birthdate" onSubmit={handleSubmit(onBirthdateSubmit)}>
+            {dobAlert && (
+              <AlertBox type={dobAlert.type} onClose={() => setDobAlert(null)} inverted closeable>
+                {dobAlert.message}
+              </AlertBox>
+            )}
+            <div className="form-card__group border-b">
+              <label className="field-label--caps" htmlFor="birthMonth">
+                {`${t("application.name.yourDateOfBirth")}`}
+              </label>
+              <div className="field-group--dob mt-2">
+                <Field
+                  name="birthMonth"
+                  placeholder={`${t("account.settings.placeholders.month")}`}
+                  error={errors.birthMonth}
+                  register={register}
+                  defaultValue={profile ? moment(new Date(profile.dob)).utc().format("MM") : null}
+                />
+                <Field
+                  name="birthDay"
+                  placeholder={`${t("account.settings.placeholders.day")}`}
+                  error={errors.birthDay}
+                  register={register}
+                  defaultValue={profile ? moment(new Date(profile.dob)).utc().format("DD") : null}
+                />
+                <Field
+                  name="birthYear"
+                  placeholder={`${t("account.settings.placeholders.year")}`}
+                  error={errors.birthYear}
+                  register={register}
+                  defaultValue={profile ? moment(new Date(profile.dob)).utc().format("YYYY") : null}
+                />
+              </div>
+              <div className="text-center mt-5">
+                <Button
+                  onClick={() => {
+                    //
+                  }}
```

Comment:
This case is in a few places in the application - I think we should avoid using an empty function, instead of we can create a condition in the <Button /> component (most of the button usage cases don't require an onClick event.